common/utils/src/main/resources/error/error-classes.json

```diff
@@ -314,6 +314,18 @@
       "<details>"
     ]
   },
+  "CANNOT_WRITE_STATE_FILE" : {
```

Comment:
How about considering symmetric name? For the base error class of failure on loading state, we use CANNOT_LOAD_STATE_STORE. Probably better to sync between two.